001-hello-pipecat-nim-video-test.ipynb

```diff
@@ -0,0 +1,455 @@
+{
```

Comment:
Super tiny Nit: maybe we should call this file `001-hello-pipecat-nim-video.ipynb` (no -test) since we don't have -test in the other file.

Reply:
ah, this file was checked in by mistake. 👍